app/src/main/java/ar/com/wolox/android/example/ui/news/NewsViewHolderAdapter.kt

```diff
@@ -59,9 +61,13 @@ class NewsViewHolderAdapter : RecyclerView.Adapter<NewsViewHolderAdapter.ViewHol
         var vNewsImage = item.findViewById<SimpleDraweeView>(R.id.vNewsImage)
         var vFavButton = item.findViewById<ToggleButton>(R.id.vFavButton)
     }
+}
 
-    companion object {
-        private const val LIKE_PRESSED = "pressed"
-        private const val LIKE_NON_PRESSED = "nonpressed"
+class NewsAdapterCallBack : DiffUtil.ItemCallback<News>() {
+    override fun areItemsTheSame(oldItem: News, newItem: News): Boolean {
+        return oldItem.id == newItem.id
+    }
+    override fun areContentsTheSame(oldItem: News, newItem: News): Boolean {
+        return (oldItem.title == newItem.title && oldItem.text == newItem.text)
```

Comment:
We have to compare all items to determine if contents are the same, you have to do `oldItem == newItem`